packages/v4/patternfly-docs/content/accessibility/alert-group/alert-group.md

```diff
@@ -3,54 +3,86 @@ id: Alert group
+At a minimumm, an alert group should meet the following criteria:
 
-- **Aria-relevant**- The aria-relevant=[LIST_OF_CHANGES] sets what types of changes are relevant to a live region and what qualifies as an update to it. The possible settings are one or more of: additions, removals, text, all. The default setting is: additions text.
+<List isPlain>
+  <ListItem>
+    <Checkbox id="alertGroup-a11y-checkbox-1" label="The alert group exists on page load and is not dynamically rendered." description="This should always exist in the DOM, especially when alerts will dynamically appear or update within it." />
+  </ListItem>
+  <ListItem>
+    <Checkbox id="alertGroup-a11y-checkbox-2" label={<span>If alerts will dynamically appear or update, the alert group has the <code class="ws-code">aria-live="polite"</code> attribute.</span>} description="This will allow assistive technologies to announce dynamically rendered alerts." />
+  </ListItem>
+</List>
 
+## React customization
 
+Various React props have been provided for more fine-tuned control over accessibility.
 
-To further customize your alert group's accessibility, you can edit the following:
+| Prop | Applied to | Reason | 
+|---|---|---|
+| `isLiveRegion` | `AlertGroup` | Makes the alert group a live region by setting `aria-live="polite"` and `aria-atomic="false"`. Instead of passing this prop in, you can manually set `aria-live` and `aria-atomic` by passing them in individually. Pass this prop in if you intend or expect the contents of the alert group to be dynamically updated. |
+| `isToast` | `AlertGroup` | Adds styling to position alerts in the top-right corner of the viewport. When passing this prop in, the `isLiveRegion` prop must also be passed in. For more information about accessibility with toast alerts, read the [toast alerts](/components/alert/accessibility#toast-alerts) section of the alert accessibility tab. |
+| `overflowMessage` | `AlertGroup` | When functionality is added for overflowing alerts, adds a custom message for the hidden overflow. When passing this prop in the message should include how many alerts are currently within the overflow and should update as alerts are added or removed. This will allow users of assistive technologies to be notified when an alert is added to the overflow, as otherwise the addition of the alert and the alert contents itself will not be announced to them. <br/><br/> When passing this prop in, the `isLiveRegion` prop must also be passed in. For more information about alert overflow, read [alert group - managing overflow](/components/alert-group/design-guidelines#managing-overflow). |
```

Comment:
This is a bit nit-picky but could we re-write this a tiny bit to say "When functionality is added for overflowing alerts, **this prop** adds a custom message for the hidden overflow" it just reads a little weird otherwise haha

Reply:
Good catch! It does read a little weird; I also updated a line in the alert file that had some similar verbiage

Reply:
Sorry last thing but could you add a coma haha " When passing the prop in, the message...."